### PR TITLE
How to describe single parent-child relationships

### PR DESCRIPTION
**Pull request by tychonievich**
As recommended in https://github.com/FamilySearch/GEDCOM/issues/334

### CODE REVIEW

File: _pages/techfaqs.md [hunk 1]
```diff
@@ -253,3 +253,40 @@ The above example denotes that the second form is the actual birth name, but the
...
+
+Certain additional information about specific aspects of a parent-child relationship can be provided in other structures:
+
+- `ADOP`.`FAMC` indicates the parents involved in a child's adoption.
+
+- `ASSO` with a family relationship `ROLE` -- including like `CHIL`, `FATH`, `MOTH`, and `PARENT` -- is for indicating a person acting in that role in a non-family context, such inside a `WILL` indicating relationships the will described for its beneficiaries or inside a `MARR` indicating guests who were present in the capacity of parents.
+
+- `BIRT`.`FAMC` indicates the parents as pertaining to a child's birth.
+
+- `SLGC`.`FAMC` indicates a potentially-posthumous recognition of a persons parents by the Church of Jesus Christ of Latter-Day Saints
```
Comment (tychonievich) [inline _pages/techfaqs.md:292]:
```suggestion
- `SLGC`.`FAMC` indicates a recognition of a persons parents by the Church of Jesus Christ of Latter-Day Saints.
```

File: _pages/techfaqs.md [hunk 2]
```diff
@@ -253,3 +253,40 @@ The above example denotes that the second form is the actual birth name, but the
 the person.  Although the specification does not define any semantics to the order among NAME structures,
 some implementations are known to infer that the first NAME occuring is a "preferred" one to use
 for display.
+
+
+# How do I record information about one parent?
+
+Parent-child relationships are stored in the `INDI`.`FAMC` structure, which must point to a `FAM` not an `INDI`.
+Additional details about that relationship are stored in its `PEDI` substructure.
+To describe the relationship of an `INDI` to just one parent, make a `FAM` that has only one partner.
```
[the remark this answers is not in the document]
Comment (tychonievich) [reply _pages/techfaqs.md:262]: Also recommend against duplicating relationships with the same partners

[the remark this answers is not in the document]
Comment (Norwegian-Sardines) [reply _pages/techfaqs.md:262]: I'm not sure I understand:

> Add that there is risk of misinterpretation, with recommendation the second `FAM` should not be assumed to mean another `MARR`.

Every addition of a `FAM` record indicates a new relationship between two adults, even if only one adult is known.  It does not (should not) assume that a marriage occurred.  People (MM, FF or MF) living together can form a "family" as can two estranged individuals.  All of these combinations can produce children either biologically, via adoption, fostering, or sealing. The sex or gender of the adult participants in the `FAM` can only be determined by accessing the actual record for that individual (`INDI`) via the WIFE/HUSB links, which contains all other information needed to determine the current gender of the individual AND more importantly the preferred name and any restrictions for displaying (privacy) of data such as birth, death via the event restriction tag.  Programs that don't follow these protocols are at risk of violating date security constrains.

[the remark this answers is not in the document]
Comment (dthaler) [reply _pages/techfaqs.md:262]: > Every addition of a `FAM` record indicates a new relationship between two adults, even if only one adult is known.

I agree with that, and don't think we should recommend creating a new FAM structure just to hold details about the relationship to one of the two parents in another FAM structure.

File: _pages/techfaqs.md [hunk 3]
```diff
@@ -253,3 +253,40 @@ The above example denotes that the second form is the actual birth name, but the
...
+1 NOTE The family as it appeared later
+1 CHIL @I1@
+1 WIFE @I2@
+1 HUSB @I3@
+0 @I1@ INDI
+1 FAMC @F1@
+2 NOTE The socially-accepted family
+1 FAMC @F2@
+2 NOTE The couple who produced the child
+0 @F2@
```
Comment (tychonievich) [inline _pages/techfaqs.md:278]:
```suggestion
0 @F2@ FAM
```

File: _pages/techfaqs.md [hunk 4]
```diff
@@ -253,3 +253,40 @@ The above example denotes that the second form is the actual birth name, but the
...
+0 @F1@ FAM
+1 NOTE The family as it appeared later
+1 CHIL @I1@
+1 WIFE @I2@
+1 HUSB @I3@
+0 @I1@ INDI
+1 FAMC @F1@
+2 NOTE The socially-accepted family
+1 FAMC @F2@
+2 NOTE The couple who produced the child
```
Comment (tychonievich) [inline _pages/techfaqs.md:277]:
```suggestion
1 FAMC @F1@
1 FAMC @F2@
```

**Comment by dthaler** [comment]
Approved during GEDCOM Steering Committee meeting.  Any other discussion will be handled under another issue.

**Pull request merged**